src/preprocessing/preprocessing_pass_registry.cpp

```diff
@@ -89,7 +90,7 @@ PreprocessingPass* PreprocessingPassRegistry::createPass(
 
 std::vector<std::string> PreprocessingPassRegistry::getAvailablePasses()
 {
-  std::vector<std::string> passes;
+  thread_local std::vector<std::string> passes;
```

Comment:
Why should this be thread_local?

Reply:
It was flagged with `-fsanitize=thread`, but it may be redundant now that the pointer for the PreprocessingPassRegistry is thread_local. I'll check.

Reply:
Seems okay now, I'll update it.